installer/lib/phx_new/generator.ex

```diff
@@ -150,12 +151,14 @@ defmodule Phx.New.Generator do
       phoenix_static_path: phoenix_static_path(phoenix_path),
       pubsub_server: pubsub_server,
       secret_key_base: random_string(64),
+      live_view_signing_salt: random_string(64),
```

Comment:
So I had already added this in the past, see `lv_signing_salt` two lines below. :)